site/content/component/auth-server.md

```diff
@@ -127,5 +127,5 @@ The corresponding command to start up the server with the configuration used in
 
 {{% note %}}
 You may want to make logging of the server a little more verbose by enabling the *dev* Spring profile.
-To do so, append *,--spring.profiles.active=authentication-impl,dev* to the command line.
+To do so, append *-Dspring.profiles.active=authentication-impl,dev* to the command line.
```

Comment:
Actually both versions are working:
-Dspring.profiles.active=... and
-Drun.profiles=...

I change it to the "run.profiles" since you are right - the documentation of the plugin uses this key, so it is better that we use it as well of course.